test/integration-v2.sh

```diff
@@ -9,41 +9,41 @@ set -euo pipefail
 result=1
```

Comment:
I would at some point rewrite it to https://github.com/thanos-io/thanos/blob/master/test/e2e/e2ethanos/services.go#L154

Reply:
yeah, I've been thinking about this for a while; this repo has a lot of space for refactoring, both the tests and the actual packages. These recent changes will at least unblock us to keep cleaning up